packages/actor-dereference-file/test/ActorDereferenceFile-test.ts

```diff
@@ -69,7 +70,22 @@ describe('ActorDereferenceFile', () => {
         {
           data: expect.anything(),
           exists: true,
-          url: p,
+          url: `file://${p}`,
+        },
+      );
+    });
+
+    it('should run for relative paths', async() => {
+      // Relative paths are considered to start from the directory that `node` is ran in
```

Comment:
We can not always guarantee this. I recommend using `__dirname` instead, and navigating from there.

Reply:
I'm not sure I get it..
Thing is that current implementation (`pathToFileURL`) resolves relative paths from the directory that node is ran in,
this UX is according to user's expectations,
say they run
```
/current/dir/$ ./bin/query.js ./file.jsonld ...
```
Then they would expect `file.jsonld` to be resolved to `/current/dir/file.jsonld`

Same happens when we run tests from `/path/to/comunica`, relative paths get resolved relative to that dir, and not a dir that a currently executed file is in (in our case, `/path/to/comunica/packages/actor-dereference-file/test/`), so, in order to that we handle relative file paths corectly, we're in need to pass in a relative path that starts from `/path/to/comunica`, which's achieved by a bit tricky to grasp following code line

My first intent been to use `__dirname` in tests to point to the asset file
I expected `path.join(__dirname, "./assets/dummy.jsonld")` to resolve to
`file:///path/to/comunica/packages/actor-dereference-file/test/assets/dummy.jsonld`
whereas, to my surprise, it's been getting resolved to
`file:///path/to/comunica/assets/dummy.jsonld`
That led me to realization, that in order to test current implementation correctly we'd need to craft relative paths from the root of the repo, where tests were launched from

Please let me know if understood your concerns right about this trickery or there is something else you meant to point onto

Reply:
Ok, that makes sense.
Could you just double-check that the test also still passes if it is run from another directory? (I suspect it will)

If so, ping me if this PR is ready for another review.

Reply:
Sure thing!
Just ran a test - test passes no prob.


Tested that it works by running
```
/path/to/comunica/packages/actor-dereference-file/test/$ npx jest@27.5.1 -c ../../jest.config.js
```

PR seems ready for having another look